src/airbash

```diff
@@ -8,41 +7,79 @@
 # access point, thus forcing a reconnect. In order to capture the handshake,
 # airodump is being run after sending the deauthentication packet. Verification
 # of a captured handshake is done using aircrack-ng
+
+# get platform
+if [ -d "/system" ]; then
+    device=0  # android
+else
+    device=1  # linux
+fi
+
 airocheck() {
 	# checks a handshake (passed by file name) using aircrack-ng
 	# returns 1 on success and 0 on failure
 	for s in `ls "$1"*.cap 2>/dev/null`; do
-		if [ `aircrack-ng -a 2 "$s" -w "$path$wl".c 2>/dev/null | grep -c valid` -eq 0 ]; then
-			if [ `aircrack-ng -a 2 "$s" -w "$path$wl".c 2>/dev/null | awk '{print $NF}' | grep -c handshake` -ne 0 ]; then
+		if [ `"$AIRCRACK_BIN" -a 2 "$s" -w "$path$wl"known.txt 2>/dev/null | grep -c valid` -eq 0 ]; then
+			if [ `"$AIRCRACK_BIN" -a 2 "$s" -w "$path$wl"known.txt 2>/dev/null | awk '{print $NF}' | grep -c handshake` -ne 0 ]; then
 				cs=1
 			fi
 		fi
 	done
 }
 airekill() {
 	# kill all instances of aireplay-ng to prevent locking
-	for a in `pgrep aireplay-ng`; do
+	for a in `pgrep $(basename "$AIREPLAY_BIN")`; do
 		kill -9 $a &>/dev/null
 	done
 }
 airokill() {
 	# kill all instances of airodump-ng to prevent locking
-	for a in `pgrep airodump-ng`; do
+	for a in `pgrep $(basename "$AIRODUMP_BIN")`; do
 		kill -9 $a &>/dev/null
 	done
 }
 
-# configure path variables
-path=/path/to/airbash-folder/
-db=.db.sqlite3
-bl=.bl
-hs=.hs/
-loc=.loc/
-wl=.wl/
+# enviro stuff
+WDIR=`pwd`  # android
+#WDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"  # linux
```

Comment:
This line will trigger an error on POSIX shells, preventing the script to run. If you run this script on bash, it is recommended to uncomment this line (and comment out the previous one).